templates/cicd/manifest.yml

```diff
@@ -0,0 +1,23 @@
+# This manifest defines the relationship and deployment ordering of your environments.
+
+# The version of the schema
+version: {{.Version}}
+
+# This section defines the source artifacts.
+source:
+  # The name of the provider that is used to store the source artifacts.
+  provider: {{.Provider}}
+  # Additional properties that further specifies the exact location
+  # the artifacts should be sourced from.
+  properties:
+    # Repository in the format of <user_name>/<repository_name>.
+    repository: {{.Repository}}
+
```

Comment:
I thought about it a little bit and decided to leave it out for now since app manifests already contain the docker file paths and archer is supposed to be able to generate CF templates based on application manifests so maybe it's possible for customers to not specify anything at all for the build stage?